### PR TITLE
Update tvtv.us_us-local.channels.xml

### PR DESCRIPTION
**Pull request by TheDeatCrusher**

### CODE REVIEW

File: sites/tvtv.us/tvtv.us_us-local.channels.xml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <site site="tvtv.us">
   <channels>
+    <channel lang="en" xmltv_id="WTNZ.us" site_id="11602">FOX (WTNZ) KNOXVILLE</channeL>
```
Comment (freearhey) [inline sites/tvtv.us/tvtv.us_us-local.channels.xml:4]:
```suggestion
    <channel lang="en" xmltv_id="WTNZ.us" site_id="11602">FOX (WTNZ) KNOXVILLE</channel>
```

File: sites/tvtv.us/tvtv.us_us-local.channels.xml [hunk 2]
```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <site site="tvtv.us">
   <channels>
+    <channel lang="en" xmltv_id="WTNZ.us" site_id="11602">FOX (WTNZ) KNOXVILLE</channeL>
+    <channel lang="en" xmltv_id="WBIR.us" site_id="11305">NBC (WBIR) KNOXVILLE</channeL>
```
Comment (freearhey) [inline sites/tvtv.us/tvtv.us_us-local.channels.xml:5]:
```suggestion
    <channel lang="en" xmltv_id="WBIR.us" site_id="11305">NBC (WBIR) KNOXVILLE</channel>
```

**Review by freearhey — CHANGES_REQUESTED** [review]
Also please make sure that the `xmltv_id` of the channels is the same as here: https://iptv-org.github.io

File: sites/tvtv.us/tvtv.us_us-local.channels.xml (hunk 1 above)
Comment (TheDeatCrusher) [reply sites/tvtv.us/tvtv.us_us-local.channels.xml]: I made another commit and fixed it

File: sites/tvtv.us/tvtv.us_us-local.channels.xml [hunk 3]
```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <site site="tvtv.us">
   <channels>
+    <channel lang="en" xmltv_id="WTNZDT1.us" site_id="11602">FOX (WTNZ) KNOXVILLE</channeL>
```
Comment (freearhey) [inline sites/tvtv.us/tvtv.us_us-local.channels.xml:4]:
```suggestion
    <channel lang="en" xmltv_id="WTNZDT1.us" site_id="11602">FOX (WTNZ) KNOXVILLE</channel>
```

File: sites/tvtv.us/tvtv.us_us-local.channels.xml [hunk 4]
```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <site site="tvtv.us">
   <channels>
+    <channel lang="en" xmltv_id="WTNZDT1.us" site_id="11602">FOX (WTNZ) KNOXVILLE</channeL>
+    <channel lang="en" xmltv_id="WBIRDT1.us" site_id="11305">NBC (WBIR) KNOXVILLE</channeL>
```
Comment (freearhey) [inline sites/tvtv.us/tvtv.us_us-local.channels.xml:5]:
```suggestion
    <channel lang="en" xmltv_id="WBIRDT1.us" site_id="11305">NBC (WBIR) KNOXVILLE</channel>
```

**Review by freearhey — CHANGES_REQUESTED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**